docs/contracts/visibility-and-getters.rst

```diff
@@ -24,27 +44,20 @@ For state variables, ``external`` is not possible.
 
 ``public``
     Public functions are part of the contract interface
-    and can be either called internally or via
-    messages. For public state variables, an automatic getter
-    function (see below) is generated.
+    and can be either called internally or via message calls.
 
 ``internal``
-    Those functions and state variables can only be
-    accessed internally (i.e. from within the current contract
-    or contracts deriving from it), without using ``this``.
-    This is the default visibility level for state variables.
+    Internal functions can only be accessed from within the current contract
+    or contracts deriving from it.
+    They cannot be accessed externally.
+    They can use internal types like mappings as parameters.
 
 ``private``
-    Private functions and state variables are only
-    visible for the contract they are defined in and not in
-    derived contracts.
-
-.. note::
-    Everything that is inside a contract is visible to
-    all observers external to the blockchain. Making something ``private``
-    only prevents other contracts from reading or modifying
-    the information, but it will still be visible to the
-    whole world outside of the blockchain.
+    Private functions are only visible for the contract they are defined in and not in derived contracts.
+    They cannot be accessed externally.
```

Comment:
Update to match https://github.com/ethereum/solidity/pull/11999/files#r744680506.
```suggestion
    They cannot be accessed externally and can use internal types for parameters.
```
Maybe it would be better to just say that they're like ``internal`` ones but not visible in derived contracts to avoid repeating stuff.